### ISSUE TITLE
Error pip installing 3.13.1

### ISSUE THREAD

**Issue by rmjarvis** [timj/starlink-pyast#31]
I'm getting an error installing pyast version 3.13.1:
```
$ pip install -U starlink-pyast
Collecting starlink-pyast
  Downloading https://files.pythonhosted.org/packages/e2/ff/7354165c9fde015d887381aba11ead0f66b64c30bb38b0aac5d501480fea/starlink-pyast-3.13.1.tar.gz (8.7MB)
     |████████████████████████████████| 8.7MB 4.3MB/s 
Building wheels for collected packages: starlink-pyast
  Building wheel for starlink-pyast (setup.py) ... error
  ERROR: Command errored out with exit status 1:
   command: /anaconda3/envs/py3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/h8/0h3_h20d4jdg8pk8fzwxmr380000gn/T/pip-install-31az_f5w/starlink-pyast/setup.py'"'"'; __file__='"'"'/private/var/folders/h8/0h3_h20d4jdg8pk8fzwxmr380000gn/T/pip-install-31az_f5w/starlink-pyast/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /private/var/folders/h8/0h3_h20d4jdg8pk8fzwxmr380000gn/T/pip-wheel-bubju5jk --python-tag cp36
       cwd: /private/var/folders/h8/0h3_h20d4jdg8pk8fzwxmr380000gn/T/pip-install-31az_f5w/starlink-pyast/
  Complete output (35 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.macosx-10.7-x86_64-3.6
  creating build/lib.macosx-10.7-x86_64-3.6/starlink
  copying starlink/__init__.py -> build/lib.macosx-10.7-x86_64-3.6/starlink
  copying starlink/Grf.py -> build/lib.macosx-10.7-x86_64-3.6/starlink
  copying starlink/Atl.py -> build/lib.macosx-10.7-x86_64-3.6/starlink
  creating build/lib.macosx-10.7-x86_64-3.6/starlink/include
  creating build/lib.macosx-10.7-x86_64-3.6/starlink/include/star
  copying starlink/include/star/pyast.h -> build/lib.macosx-10.7-x86_64-3.6/starlink/include/star
  running build_ext
  building 'starlink.Ast' extension
  creating build/temp.macosx-10.7-x86_64-3.6
  creating build/temp.macosx-10.7-x86_64-3.6/starlink
  creating build/temp.macosx-10.7-x86_64-3.6/starlink/ast
  creating build/temp.macosx-10.7-x86_64-3.6/ast
  creating build/temp.macosx-10.7-x86_64-3.6/ast/src
  creating build/temp.macosx-10.7-x86_64-3.6/ast/wcslib
  creating build/temp.macosx-10.7-x86_64-3.6/ast/cminpack
  creating build/temp.macosx-10.7-x86_64-3.6/ast/erfa
  gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/anaconda3/envs/py3/include -arch x86_64 -I/anaconda3/envs/py3/include -arch x86_64 -DHAVE_STRTOK_R=1 -DHAVE_STRERROR_R=1 -DHAVE_DECL_ISFINITE=1 -DSIZEOF_LONG=8 -DSIZEOF_LONG_LONG=8 -DHAVE_DECL_ISNAN=1 -I/anaconda3/envs/py3/lib/python3.6/site-packages/numpy/core/include -I./starlink/include -I./ast -I./ast/src -I/anaconda3/envs/py3/include/python3.6m -c starlink/ast/Ast.c -o build/temp.macosx-10.7-x86_64-3.6/starlink/ast/Ast.o
  In file included from /anaconda3/envs/py3/lib/python3.6/site-packages/numpy/core/include/numpy/ndarraytypes.h:1830:0,
                   from /anaconda3/envs/py3/lib/python3.6/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                   from /anaconda3/envs/py3/lib/python3.6/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                   from starlink/ast/Ast.c:3:
  /anaconda3/envs/py3/lib/python3.6/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
   #warning "Using deprecated NumPy API, disable it with " \
    ^
  starlink/ast/Ast.c:6:21: fatal error: src/grf.h: No such file or directory
   #include "src/grf.h"
                       ^
  compilation terminated.
  error: command 'gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for starlink-pyast
  Running setup.py clean for starlink-pyast
Failed to build starlink-pyast
Installing collected packages: starlink-pyast
  Found existing installation: starlink-pyast 3.10.0
    Uninstalling starlink-pyast-3.10.0:
      Successfully uninstalled starlink-pyast-3.10.0
  Running setup.py install for starlink-pyast ... error
    ERROR: Command errored out with exit status 1:
     command: /anaconda3/envs/py3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/h8/0h3_h20d4jdg8pk8fzwxmr380000gn/T/pip-install-31az_f5w/starlink-pyast/setup.py'"'"'; __file__='"'"'/private/var/folders/h8/0h3_h20d4jdg8pk8fzwxmr380000gn/T/pip-install-31az_f5w/starlink-pyast/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/h8/0h3_h20d4jdg8pk8fzwxmr380000gn/T/pip-record-d761q4ev/install-record.txt --single-version-externally-managed --compile
         cwd: /private/var/folders/h8/0h3_h20d4jdg8pk8fzwxmr380000gn/T/pip-install-31az_f5w/starlink-pyast/
    Complete output (35 lines):
    running install
    running build
    running build_py
    creating build
    creating build/lib.macosx-10.7-x86_64-3.6
    creating build/lib.macosx-10.7-x86_64-3.6/starlink
    copying starlink/__init__.py -> build/lib.macosx-10.7-x86_64-3.6/starlink
    copying starlink/Grf.py -> build/lib.macosx-10.7-x86_64-3.6/starlink
    copying starlink/Atl.py -> build/lib.macosx-10.7-x86_64-3.6/starlink
    creating build/lib.macosx-10.7-x86_64-3.6/starlink/include
    creating build/lib.macosx-10.7-x86_64-3.6/starlink/include/star
    copying starlink/include/star/pyast.h -> build/lib.macosx-10.7-x86_64-3.6/starlink/include/star
    running build_ext
    building 'starlink.Ast' extension
    creating build/temp.macosx-10.7-x86_64-3.6
    creating build/temp.macosx-10.7-x86_64-3.6/starlink
    creating build/temp.macosx-10.7-x86_64-3.6/starlink/ast
    creating build/temp.macosx-10.7-x86_64-3.6/ast
    creating build/temp.macosx-10.7-x86_64-3.6/ast/src
    creating build/temp.macosx-10.7-x86_64-3.6/ast/wcslib
    creating build/temp.macosx-10.7-x86_64-3.6/ast/cminpack
    creating build/temp.macosx-10.7-x86_64-3.6/ast/erfa
    gcc -Wno-unused-result -Wsign-compare -Wunreachable-code -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -I/anaconda3/envs/py3/include -arch x86_64 -I/anaconda3/envs/py3/include -arch x86_64 -DHAVE_STRTOK_R=1 -DHAVE_STRERROR_R=1 -DHAVE_DECL_ISFINITE=1 -DSIZEOF_LONG=8 -DSIZEOF_LONG_LONG=8 -DHAVE_DECL_ISNAN=1 -I/anaconda3/envs/py3/lib/python3.6/site-packages/numpy/core/include -I./starlink/include -I./ast -I./ast/src -I/anaconda3/envs/py3/include/python3.6m -c starlink/ast/Ast.c -o build/temp.macosx-10.7-x86_64-3.6/starlink/ast/Ast.o
    In file included from /anaconda3/envs/py3/lib/python3.6/site-packages/numpy/core/include/numpy/ndarraytypes.h:1830:0,
                     from /anaconda3/envs/py3/lib/python3.6/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                     from /anaconda3/envs/py3/lib/python3.6/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                     from starlink/ast/Ast.c:3:
    /anaconda3/envs/py3/lib/python3.6/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
     #warning "Using deprecated NumPy API, disable it with " \
      ^
    starlink/ast/Ast.c:6:21: fatal error: src/grf.h: No such file or directory
     #include "src/grf.h"
                         ^
    compilation terminated.
    error: command 'gcc' failed with exit status 1
    ----------------------------------------
  Rolling back uninstall of starlink-pyast
  Moving to /anaconda3/envs/py3/lib/python3.6/site-packages/starlink/
   from /anaconda3/envs/py3/lib/python3.6/site-packages/~tarlink
  Moving to /anaconda3/envs/py3/lib/python3.6/site-packages/starlink_pyast-3.10.0.dist-info/
   from /anaconda3/envs/py3/lib/python3.6/site-packages/~tarlink_pyast-3.10.0.dist-info
ERROR: Command errored out with exit status 1: /anaconda3/envs/py3/bin/python -u -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/h8/0h3_h20d4jdg8pk8fzwxmr380000gn/T/pip-install-31az_f5w/starlink-pyast/setup.py'"'"'; __file__='"'"'/private/var/folders/h8/0h3_h20d4jdg8pk8fzwxmr380000gn/T/pip-install-31az_f5w/starlink-pyast/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /private/var/folders/h8/0h3_h20d4jdg8pk8fzwxmr380000gn/T/pip-record-d761q4ev/install-record.txt --single-version-externally-managed --compile Check the logs for full command output.
```
Version 3.12 works fine:
```
$ pip install starlink-pyast==3.12
Collecting starlink-pyast==3.12
  Downloading https://files.pythonhosted.org/packages/61/7f/1ed7c5e30f848f284e5320670cf7b7a71b665ac8270fb9b65f3e823436d9/starlink-pyast-3.12.0.tar.gz (9.9MB)
     |████████████████████████████████| 9.9MB 3.9MB/s 
Building wheels for collected packages: starlink-pyast
  Building wheel for starlink-pyast (setup.py) ... done
  Created wheel for starlink-pyast: filename=starlink_pyast-3.12.0-cp36-cp36m-macosx_10_7_x86_64.whl size=2979793 sha256=bd0ce494c1980db6cb4929baef9e92e742bd10d923dfc99e8cb5a1a9f378ed13
  Stored in directory: /Users/Mike/Library/Caches/pip/wheels/cf/3d/0a/32ef9a22fc328b81fce1dd312f691b0ff64a6de6ced121017b
Successfully built starlink-pyast
Installing collected packages: starlink-pyast
  Found existing installation: starlink-pyast 3.10.0
    Uninstalling starlink-pyast-3.10.0:
      Successfully uninstalled starlink-pyast-3.10.0
Successfully installed starlink-pyast-3.12.0
```

**Comment by dsberry** [comment]
I forgot to update the manifest. Should be fixed in pyast 3.13.2.

**Comment by rmjarvis** [comment]
Yes.  Works now.  Thanks!
